BigBaseV2/src/views/view_weapons.cpp

```diff
@@ -1,5 +1,9 @@
 #include "core/data/custom_weapons.hpp"
 #include "views/view.hpp"
+#include "script.hpp"
+#include "gta/Weapons.h"
+
+#include "fiber_pool.hpp"
```

Comment:
Set these alphabetically, to follow the same coding style as the entire project.